src/software/backend/grsim_backend.cpp

```diff
@@ -10,9 +11,11 @@ GrSimBackend::GrSimBackend()
                     Util::Constants::SSL_VISION_MULTICAST_PORT,
                     Util::Constants::SSL_GAMECONTROLLER_MULTICAST_ADDRESS,
                     Util::Constants::SSL_GAMECONTROLLER_MULTICAST_PORT,
-                    boost::bind(&GrSimBackend::receiveWorld, this, _1)),
+                    boost::bind(&GrSimBackend::receiveWorld, this, _1),
+                    Util::DynamicParameters->getAIControlConfig()->getRefboxConfig()),
```

Comment:
A note to anyone reviewing: there's some trickyness with the factories here that we will need to figure out, closely related to https://github.com/UBC-Thunderbots/Software/pull/1163#issuecomment-586733276